### PR TITLE
Fixing formatting of script output

### LINKED ISSUE #54: Mac documentation formatting
I noticed the formatting for the `docker run hello-world` example in the Mac instructions (step_one.md) were not aligned. This aligns the output so it renders like the Linux and Windows docs.

### PR DESCRIPTION
**Pull request by nloadholtes**
I noticed the formatting for the `docker run hello-world` example in the Mac instructions (step_one.md) were not aligned. This aligns the output so it renders like the Linux and Windows docs.

Fixes #54

### CODE REVIEW

**Comment by SvenDowideit** [comment]
LGTM - @thaJeztah @moxiegirl

**Comment by thaJeztah** [comment]
LGTM, thanks!

(CI is being scheduled, @SvenDowideit do we need to wait for that?)

**Comment by thaJeztah** [comment]
(we miss a "status/merge" label btw)

**Comment by moxiegirl** [comment]
@SvenDowideit what is the status on the CI here?

**Comment by thaJeztah** [comment]
test are green now, merging \o/

**Comment by thaJeztah** [comment]
oh, hold on; there's an extra commit added. also, I just spotted that the indentation is using tabs, not spaces.

@nloadholtes can you;

- squash your commits, so that there's only one commit in this pr
- use 4 spaces in stead of a tab for indentation?

**Comment by nloadholtes** [comment]
Good catch on the tabs! I can't believe I didn't notice that.

Can you offer any advice on why the documentation check is red? It looks like git thinks its a detached head, but I've pulled from upstream, rebased, pushed to my repo, then squashed.

**Comment by thaJeztah** [comment]
@nloadholtes good question, no idea actually :blush:

possibly @SvenDowideit knows

File: docs/mac/step_one.md [hunk 1]
```diff
@@ -148,41 +148,35 @@ Virtual Box VM, it maintains its configuration between uses.
...
+        Share images, automate workflows, and more with a free Docker Hub account:
+        https://hub.docker.com
 
         For more examples and ideas, visit:
-         https://docs.docker.com/userguide/
+        https://docs.docker.com/userguide/
 
 
-## Where to go next
+## Where to go nextxs
```
Comment (thaJeztah) [inline docs/mac/step_one.md]: unrelated change?

Comment (nloadholtes) [inline docs/mac/step_one.md]: Whoops, I'll clean that up. This is one of those moments where we need a facepalm emoji

Comment (thaJeztah) [inline docs/mac/step_one.md]: haha, no problem

**Comment by moxiegirl** [comment]
@nloadholtes Thank you for the contribution!

**Comment by nloadholtes** [comment]
Thanks!
